docs/conf.py

```diff
@@ -135,8 +159,8 @@ def noop_decorator(*args, **kwargs):
 # default: ``['localtoc.html', 'relations.html', 'sourcelink.html',
 # 'searchbox.html']``.
 #
-html_sidebars = {
-    "**": ["globaltoc.html", "relations.html", "sourcelink.html", "searchbox.html"]
-}
+# html_sidebars = {
```

Comment:
I commented this out because the default html_sidebars for pydata_sphinx_theme is sufficient. See [docs](https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/configuring.html#configure-the-sidebar). Another way of saying it is these html_sidebars don't look good on the new theme. I've removed the html_sidebars comment for clarity.